dask_kubernetes/operator/deployment/helm/dask-kubernetes-operator/values.yaml

```diff
@@ -85,3 +85,7 @@ metrics:
         - dask.org/cluster-name
         - dask.org/workergroup-name
       metricRelabelings: [] # MetricRelabelConfigs to apply to samples before ingestion.
+
+workerAllocationSettings:
```

Comment:
Maybe we just call this `workerAllocation` to be in line with `kubernetes.controller.worker-allocation` in the Dask config?